client/src/components/form/Form.tsx

```diff
@@ -502,18 +521,18 @@ export function FormDialog() {
                   >
                     {documents.map((doc) => (
                       <MultiSelectItem
-                        key={doc}
-                        value={doc}
+                      key={doc._id?.toString()}
+                      value={doc._id ? doc._id.toString() : ""}
                         className={
-                          documentsForProjection.includes(doc) ||
-                            documentsForDirect.includes(doc) ||
-                            documentsForCollateral.includes(doc) ||
-                            documentsForUpdate.includes(doc)
+                          documentsForProjection.includes(doc.title) ||
+                          documentsForDirect.includes(doc.title) ||
+                          documentsForCollateral.includes(doc.title) ||
+                          documentsForUpdate.includes(doc.title)
```

Comment:
the app don't removes/gray out the used doc

Reply:
resolved